src/Resources/laravel-debugbar.css

```diff
@@ -313,10 +313,6 @@ dl.phpdebugbar-widgets-kvlist dd.phpdebugbar-widgets-value.phpdebugbar-widgets-p
     background: transparent;
 }
 
-dl.phpdebugbar-widgets-kvlist dt {
-    width: calc(25%-10px);
```

Comment:
after this was removed
![image](https://github.com/barryvdh/laravel-debugbar/assets/4933954/b9298742-09c8-4acc-b3f1-82ddbd5c0a31)
now you can no longer read the model names